src/_includes/layout.njk

```diff
@@ -147,22 +147,22 @@
           <div class="col-md-6">
             <ul class="list-inline social-buttons">
               <li class="list-inline-item">
-                <a href="https://www.facebook.com/codeconnector" target="blank">
+                <a href="https://www.facebook.com/codeconnector" target="blank" aria-label="Code Connector's Facebook navigation link">
                   <i class="fab fa-facebook-f"></i>
                 </a>
-              </li>
+              </li> 
```

Comment:
nit: Remove the extra space?

Maybe I should add something to enforce an html code style or autoformat things. 🤷

Reply:
Fixed